docs/markdown/label-series.md

```diff
@@ -0,0 +1,92 @@
+## LabelSeries:
+
+<!-- INJECT:"LabelSeriesExample" -->
+
+Sometimes you just need to write on your data, and labelSeries has your back. This simple series has a similar API as the markSeries except it adds a label property to each of the rows. This label is then rendered as part of the svg tree. This can be useful for annotating points, as above, or in labeling wedges as (as in the radial chart).
+
+```javascript
+<XYPlot width={300} height={300}>
+  <LabelSeries
+    animation
+    allowOffsetToBeReversed
+    data={data} />
+</XYPlot>
+```
+
+Each series corresponds to exactly **one** svg path. It is perfectly okay to many series to express many polygons!
```

Comment:
`to many series to express` is a bit clunky. Maybe `with many series` or `for many series,`.